### DISCUSSION TITLE
Interaction events

### DISCUSSION THREAD

**Discussion by cy920820** [rexrainbow/phaser3-rex-notes#434] [General]
After creating a sprite, setting an interactive click event can trigger it normally, but when it is moved into a container and its position changes, the interactive click event does not trigger properly

**Comment by rexrainbow** [comment]
What is the 'click' event? A [button behavior](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/button/)?  Also, please provide a simplest runnable sample for describing this issue.

**Comment by cy920820** [comment]
thanks, I found the reason, it's because the container added the 'setScrollFactor' attribute, but the elements inside the container are not set, so the click event does not take effect
